tests/ui-smoke/test_homepage.py

```diff
@@ -0,0 +1,45 @@
+"""Smoke tests for addons-frontend docker image."""
+
+import pytest
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+
+@pytest.fixture
+def firefox_options(firefox_options):
+    """Set up Firefox options."""
+    firefox_options.add_argument("-headless")
+    firefox_options.add_argument('-foreground')
+    return firefox_options
+
+
+@pytest.fixture
+def selenium(base_url, selenium):
+    """Set up selenium."""
+    # Simple wait for homepage to load
+    selenium.get(base_url)
+    WebDriverWait(selenium, 60).until(
+        EC.visibility_of_element_located(
```

Comment:
I'd like to see a smoke test that waits for [`<!-- Godzilla of browsers -->`](https://github.com/mozilla/addons-frontend/blob/550fcb2bc024f25fe9c8a1d7d0e59d63a033de9e/src/amo/pages/Home/index.js#L240) and nothing else.

I don't think it's a good idea to look for `.Hero-contents` or `.Home-RecommendedExtensions` or to check the length of elements therein because all of that might change when we [re-do the home page](https://github.com/orgs/mozilla/projects/95) this quarter.

Reply:
Hmm okay. What do you think then? I Since this is a small suite, if you do change the homepage, the tests would be very simple to update.

Reply:
We don't want to update the tests ideally. If `<!-- Godzilla of browsers -->` is displayed, then we'll know that the docker image is built correctly. The rest is already covered by unit tests and other integration tests.

Reply:
Okay cool